File: core-java-practice/gcr-codebase/programming-elements/simpleInterest.java
import java.util.*;
public class simpleInterest {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("principal amount: ");
        double p = scanner.nextDouble();
        System.out.print("rate of interest: ");
        double r = scanner.nextDouble();
        System.out.print("time period (in yrs): ");
        double t = scanner.nextDouble();
        double simpleInterest = (p * r * t) / 100;
        System.out.println("The simple interest is: " + simpleInterest);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/SimpleInterest.cs
using System;

class SimpleInterest
{
    static void Main()
    {
        // Principal amount
        double principal = 1000;
        
        // Rate of interest per annum
        double rate = 5;
        
        // Time period in years
        double time = 2;
        
        // Calculate simple interest
        double simpleInterest = (principal * rate * time) / 100;
        
        // Calculate total amount
        double totalAmount = principal + simpleInterest;
        
        Console.WriteLine($"Simple Interest: {simpleInterest}");
        Console.WriteLine($"Total Amount: {totalAmount}");
    }
}